src/arch/arm/tlb.cc

```diff
@@ -239,6 +249,8 @@ TLB::insert(const Lookup &lookup_data, TlbEntry &entry)
 
     table.insertEntry(lookup_data, victim);
 
+    table.invalidatePrev(victim);
```

Comment:
No action points for this PR: I just noticed that AssociativeCache::findVictim() calls invalidate(). I think the API should be simpler and more in line with the name of the functions: findVictim() should only find a victim, and invalidate() should do the invalidation.